docs/code-snippets/outlook-snippets.yaml

```diff
@@ -947,6 +947,92 @@ Office.Mailbox#makeEwsRequestAsync:member(1):
 
         // Process the returned response here.
     }
+Office.MailboxEnums.ActionType:enum:
+  - |-
+    // Define an insight notification message.
+    const insightMessage = {
+        type: Office.MailboxEnums.ItemNotificationMessageType.InsightMessage,
+        message: "This is an insight notification",
+        icon: "Icon.80x80",
+        actions: [{
+            actionText: "Open insight",
+            actionType: Office.MailboxEnums.ActionType.ShowTaskPane,
+            commandId: "msgComposeOpenPaneButton",
+            contextData: JSON.stringify({a: "aValue", b: "bValue"})
+        }]
+    };
+Office.MailboxEnums.AttachmentStatus:enum:
+  - |-
+    // Get an attachment that was just added to a message or appointment.
+    function myHandlerFunction(eventarg) {
+        if (eventarg.attachmentStatus === Office.MailboxEnums.AttachmentStatus.Added) {
+            const attachment = eventarg.attachmentDetails;
+            console.log("Event Fired and Attachment Added!");
+            getAttachmentContentAsync(attachment.id, options, callback);
+        }
+    }
+
+    Office.context.mailbox.item.addHandlerAsync(Office.EventType.AttachmentsChanged, myHandlerFunction, myCallback);
+Office.MailboxEnums.InfobarActionType:enum:
+  - |-
+    /*
+    * This snippet activates when a notification message is dismissed from an Outlook message or appointment.
+    * The event handler prints the custom action and notification type to the console.
+    */
+    Office.context.mailbox.item.addHandlerAsync(Office.EventType.InfobarClicked, eventHandler, callback);
+
+    function eventHandler(event) {
+        const infobarDetails = event.infobarDetails;
+
+        // Print the custom action type. Currently, "Dismiss" is the only custom action supported by the InfobarClicked event.
```

Comment:
Consider moving the line about "Dismiss" is the only supported action to the reference doc page itself (instead of the sample). This page will be more futureproof that way,